faucet.go

```diff
@@ -232,6 +118,50 @@ func cleanAndExpandPath(path string) string {
 	return filepath.Clean(os.ExpandEnv(path))
 }
 
+// Verify timelimit from determined IP and count how many time
```

Comment:
Per the [guidelines in Effective Go](https://golang.org/doc/effective_go.html#commentary), comments should start with the name of the function:

> // verifyTimeLimit will return an error if the given IP address has already performed an action within the given time limit